### PR TITLE
[DatePicker] fix default date parser converting dates to UTC implicitly

### LINKED ISSUE #25811: [Bug]: Using a date format of YYYY-MM-DD with DatePicker yields incorrect results
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (2) x64 Intel(R) Core(TM) i7-4960X CPU @ 3.60GHz
    Memory: 3.41 GB / 11.78 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Browsers:
    Chrome: 107.0.5304.110
    Firefox: /home/jason/.mozilla/firefox
Found default profile: 5tdqvi2a.default-release
Import done in 1.581 s
Mozilla Firefox 107.0
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/datepicker

## Actual Behavior
Using the very demos of Fluent UI, using the DatePicker Allowing Text Input sample, input a date in the form of YYYY-MM-DD, such as 2022-11-15. When you leave the field it is changed to November 14, the day before. If, however, you enter it as MM/DD/YYYY, such as 11/15/2022, no problem.

As a matter of fact, a user already reported this issue in #22846 but it was closed as not reproducible. This is incorrect, it is easily reproducible right here: https://developer.microsoft.com/en-us/fluentui#/controls/web/datepicker

## Expected Behavior
Hand typing a date in the form of YYYY-MM-DD should not incorrectly alter the date when you leave the field.

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by Svudec**
## Previous Behavior

Default date parser is using js native method `Date.parse()` to parse string to `Date` object. 
If `Date.parse()` gets date-only ISO 8601 string format it implicitly converts it to UTC `Date` object as described [HERE](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse#date_time_string_format). 

If the user is in the time zone that is 'behind' or 'in front of' UTC enough that it has a different date, `DatePicker` will parse the wrong date (one day before or behind the date that the user entered).

Behavior can be reproduced by changing the time zones in browser's developer tools.

## New Behavior

Default date parser checks if the inserted string is in date-only ISO 8601 format and adds time to it (12:00). Because of that, Date.parse() parses the string in `Date` object with user's local time zone.

## Related Issue(s)

* Fixes #25811

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2ae8cc79c87d510b6ad5afd01b77c6a89f5debd9'>2ae8cc79c87d510b6ad5afd01b77c6a89f5debd9</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/BaseButton-mount_base.html">1189</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/BaseButton-mount.html">1177</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Breadcrumb-mount_base.html">2985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Breadcrumb-mount.html">2967</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Checkbox-mount_base.html">2643</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Checkbox-mount.html">2667</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/CheckboxBase-mount_base.html">2387</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/CheckboxBase-mount.html">2359</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ChoiceGroup-mount_base.html">4437</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ChoiceGroup-mount.html">4444</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ComboBox-mount_base.html">1252</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ComboBox-mount.html">1238</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/CommandBar-mount_base.html">9596</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/CommandBar-mount.html">9587</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ContextualMenu-mount_base.html">10999</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ContextualMenu-mount.html">10997</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DefaultButton-mount_base.html">1376</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DefaultButton-mount.html">1374</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DetailsRow-mount_base.html">3626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DetailsRow-mount.html">3604</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DetailsRowFast-mount_base.html">3599</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DetailsRowFast-mount.html">3633</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DetailsRowNoStyles-mount_base.html">3468</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DetailsRowNoStyles-mount.html">3493</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Dialog-mount_base.html">3129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Dialog-mount.html">3131</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DocumentCardTitle-mount_base.html">572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/DocumentCardTitle-mount.html">575</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Dropdown-mount_base.html">3232</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Dropdown-mount.html">3245</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/FocusTrapZone-mount_base.html">2010</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/FocusTrapZone-mount.html">2032</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/FocusZone-mount_base.html">1990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/FocusZone-mount.html">1954</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedList-mount_base.html">1934</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedList-mount.html">2149</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedList-virtual-rerender_base.html">1122</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedList-virtual-rerender.html">1128</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1658</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedList-virtual-rerender-with-unmount.html">1667</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedListV2-mount_base.html">560</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedListV2-mount.html">570</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedListV2-virtual-rerender_base.html">542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedListV2-virtual-rerender.html">542</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">551</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/IconButton-mount_base.html">1912</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/IconButton-mount.html">1902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Label-mount_base.html">727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Label-mount.html">735</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Layer-mount_base.html">4355</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Layer-mount.html">4311</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Link-mount_base.html">817</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Link-mount.html">828</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/MenuButton-mount_base.html">1659</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/MenuButton-mount.html">1677</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/MessageBar-mount_base.html">2349</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/MessageBar-mount.html">2308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Nav-mount_base.html">3548</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Nav-mount.html">3297</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/OverflowSet-mount_base.html">1358</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/OverflowSet-mount.html">1364</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Panel-mount_base.html">2569</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Panel-mount.html">2565</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Persona-mount_base.html">1319</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Persona-mount.html">1319</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Pivot-mount_base.html">1633</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Pivot-mount.html">1629</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/PrimaryButton-mount_base.html">1533</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/PrimaryButton-mount.html">1528</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Rating-mount_base.html">6979</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Rating-mount.html">7054</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/SearchBox-mount_base.html">1512</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/SearchBox-mount.html">1523</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Shimmer-mount_base.html">2877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Shimmer-mount.html">2905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Slider-mount_base.html">2083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Slider-mount.html">2111</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/SpinButton-mount_base.html">4721</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/SpinButton-mount.html">4663</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Spinner-mount_base.html">795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Spinner-mount.html">797</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/SplitButton-mount_base.html">3107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/SplitButton-mount.html">3118</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Stack-mount_base.html">822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Stack-mount.html">829</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/StackWithIntrinsicChildren-mount_base.html">2451</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/StackWithIntrinsicChildren-mount.html">2447</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/StackWithTextChildren-mount_base.html">4896</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/StackWithTextChildren-mount.html">4905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/SwatchColorPicker-mount_base.html">10518</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/SwatchColorPicker-mount.html">10525</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/TagPicker-mount_base.html">2586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/TagPicker-mount.html">2640</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/TeachingBubble-mount_base.html">86883</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/TeachingBubble-mount.html">86909</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Text-mount_base.html">783</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Text-mount.html">778</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/TextField-mount_base.html">1598</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/TextField-mount.html">1590</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ThemeProvider-mount_base.html">1529</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ThemeProvider-mount.html">1520</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ThemeProvider-virtual-rerender_base.html">1093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ThemeProvider-virtual-rerender.html">1080</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2159</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Toggle-mount_base.html">1107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/Toggle-mount.html">1098</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/buttonNative-mount_base.html">544</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25962/perf-test/buttonNative-mount.html">541</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by spmonahan — APPROVED** [review]

File: change/@fluentui-react-d7fc6747-d247-408f-9f39-3cf594589f9f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-d7fc6747-d247-408f-9f39-3cf594589f9f.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25962&commitId=64295e27f008a8cf2e8198a98f876f804ed08eb4&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Pull request merged**